presidio-analyzer/presidio_analyzer/app.py

```diff
@@ -154,6 +155,9 @@ def analyze_command_handler(text, fields, env_grpc_port=False, grpc_port=3001):
     for field_name in fields:
         field_type = request.analyzeTemplate.fields.add()
         field_type.name = field_name
+
+    request.analyzeTemplate.language = language
```

Comment:
Coming to think about it, why do we change the request here? You should be passing the language within the request, otherwise all requests will have the default language.

Reply:
Thats right. In line 141 language default value is "en". If you send another language, it will be set in this line (159). Otherwise the default language "en" will not be modified.